guide/onboarding/introduction.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: guide
+title: Onboarding
+description: An overview of some of the concepts to consider when building onboarding experiences.
+nav_order: 2
+has_children: true
+permalink: /guide/onboarding/introduction/
+---
+
+<!--
+
+Editor's notes
+
+A brief introduction and summary of all pages in this section. The idea is that readers
+scan this page to get an overview of the section and then decide which topics to dive into.
+
+-->
+
+# Onboarding
+
+Effective onboarding can make or break someone’s first bitcoin experience. Therefore, it is important to identify your target audience when building wallet-products and design specifically with that audience in mind. 
```

Comment:
```suggestion
Onboarding, the first use, can make or break someone’s experience of a product. Therefore it is important to identify your target audience and design specifically with its context, expectations, and needs in mind. 
```

This is just a suggestion. Something doesn't flow quite right with the original text. One thing is that it's not only the Bitcoin experience, but the product experience overall. Also wanted to reduce the duplicate use of the word "audience" in the second sentence. Also wanted to clarify what onboarding is right away. Just a few ideas for you to consider.

Reply:
I prefer this actually, the use of wallet-product early on can be a bit confusing